frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx

```diff
@@ -304,11 +304,9 @@ export const UploadPVCForm: React.FC<UploadPVCFormProps> = (props) => {
               dropdownUnits={dropdownUnits}
               describedBy="request-size-help"
```

Comment:
We should remove the `describedBy` prop if it's not needed and pick a value inside the component.

Reply:
why is it not needed?

Reply:
> why is it not needed?

Because we're managing the help text inside the component now. `aria-describedby` should always reference the `id` of the help text we're adding.